webpack/test-coverage.config.js

```diff
@@ -0,0 +1,12 @@
+import _ from 'lodash';
+import { jsLoader } from './base.config';
+import testConfig from './test.config';
+
+export default _.extend({}, testConfig, {
+  module: {
+    loaders: [
+      { test: /\.js/, loader: `${jsLoader}!isparta`, exclude: /node_modules|test[\/\\]/ },
+      { test: /\.js/, loader: `${jsLoader}`, include: /test[\/\\]/ }
+    ]
```

Comment:
duplicate ?

it is already defined in test.config.js.

I presume this file can be safely removed at all ?